scripts/g.extension/testsuite/test_addons_download.py

```diff
@@ -0,0 +1,141 @@
+"""
+TEST:      test_addons_download.py
+
+AUTHOR(S): Stefan Blumentrath <stefan dot blumentrath at nina dot no)
+           based on test_addons_modules.py by
+           Vaclav Petras <wenzeslaus gmail com>
+
+PURPOSE:   Test for g.extension individual modules/extensions download
+
+COPYRIGHT: (C) 2020 Stefan Blumentrath, Vaclav Petras,
+           and by the GRASS Development Team
+
+           This program is free software under the GNU General Public
+           License (>=v2). Read the file COPYING that comes with GRASS
+           for details.
+"""
+
+import sys
+import unittest
+
+from pathlib import Path
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+from grass.gunittest.utils import silent_rmtree
+
+ms_windows = sys.platform == "win32"
+
+
+class TestModuleDownloadFromDifferentSources(TestCase):
+
+    # MS Windows install function requires absolute paths
+    install_prefix = Path("gextension_test_install_path").absolute()
+
+    files = [
+        install_prefix.joinpath("scripts", "r.example.plus"),
```

Comment:
When I was suggesting Path, I was really thinking:

```suggestion
        install_prefix / "scripts" / "r.example.plus",
```

For single line, it is shorter. I'm not sure if it is readable for multiple lines, _joinpath_ might be better when it needs a line break.

Totally optional, though.

Reply:
Was not aware of this possibility. Good to know and implemented in [b0e1215](https://github.com/OSGeo/grass/pull/1158/commits/b0e12150274791f3dd7f03c62a47245095145a15)